### ISSUE TITLE
Shrimp Vendors are missing

### ISSUE THREAD

**Issue by TrueNoir** [rathena/rathena#2093]
Can't make my own pull request but I actually fixed this using my own translated korean navigation and some information on IRO figured i would share then be a selfish bastard. 
```
//=======================================================
//Alberta
//=======================================================
aldeba_in,89,56,0	shop	Cat Paw Shrimp Trader#alb	495,515:-1,567:-1,11602:-1

//=======================================================
//Amatsu
//=======================================================
amatsu,102,122,0	shop	Cat Paw Shrimp Trader#ama	495,515:-1,567:-1,11602:-1

//=======================================================
//Ayothaya
//=======================================================
ayo_in01,15,177,0	shop	Cat Paw Shrimp Trader#ayo	495,515:-1,567:-1,11602:-1

//=======================================================
//Brasilis
//=======================================================
brasilis,249,257,0	shop	Cat Paw Shrimp Trader#bra	495,515:-1,567:-1,11602:-1

//=======================================================
// Comodo
//=======================================================
comodo,103,136,0	shop	Cat Paw Shrimp Trader#como	495,515:-1,567:-1,11602:-1

//=======================================================
// Dewata
//=======================================================
dewata,193,169,0	shop	Cat Paw Shrimp Trader#dewa	495,515:-1,567:-1,11602:-1

//=======================================================
// Einbroch
//=======================================================
einbroch,164,316,0	shop	Cat Paw Shrimp Trader#ein	495,515:-1,567:-1,11602:-1
ein_in01,194,22,0	shop	Cat Paw Shrimp Trader#ein1	495,515:-1,567:-1,11602:-1

//=======================================================
// El Dicastes
//=======================================================
dic_in01,242,119,0	shop	Cat Paw Shrimp Trader#dic	495,515:-1,567:-1,11602:-1
ecl_in01,72,86,0	shop	Cat Paw Shrimp Trader#ecl	495,515:-1,567:-1,11602:-1

//=======================================================
//Geffen
//=======================================================
geffen_in,75,140,0	shop	Cat Paw Shrimp Trader#gef	495,515:-1,567:-1,11602:-1

//=======================================================
//Gonryun
//=======================================================
gonryun,147,88,0	shop	Cat Paw Shrimp Trader#gon	495,515:-1,567:-1,11602:-1

//=======================================================
//Hugel
//=======================================================
hugel,76,165,0	shop	Cat Paw Shrimp Trader#hug	495,515:-1,567:-1,11602:-1

//=======================================================
// Izlude
//=======================================================
izlude,117,165,0	shop	Cat Paw Shrimp Trader#iz	495,515:-1,567:-1,11602:-1
izlude_a,117,165,0	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_a	495
izlude_b,117,165,0	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_b	495
izlude_c,117,165,0	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_c	495
izlude_d,117,165,0	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_d	495

//=======================================================
//Lasagna
//=======================================================
lasagna,166,117,0	shop	Cat Paw Shrimp Trader#las	495,515:-1,567:-1,11602:-1

//=======================================================
// Lighthalzen
//=======================================================
lhz_in02,29,219,0	shop	Cat Paw Shrimp Trader#lhz	495,515:-1,567:-1,11602:-1

//=======================================================
//louyang
//=======================================================
lou_in02,241,168,0	shop	Cat Paw Shrimp Trader#lou	495,515:-1,567:-1,11602:-1


//=======================================================
//Lutie
//=======================================================
xmas_in,32,38,0	shop	Cat Paw Shrimp Trader#xma	495,515:-1,567:-1,11602:-1

//=======================================================
// Mora
//=======================================================
mora,110,118,0	shop	Cat Paw Shrimp Trader#mora	495,515:-1,567:-1,11602:-1

//=======================================================
//Morocc
//=======================================================
mosk_in,30,246,0	shop	Cat Paw Shrimp Trader#mosk	495,515:-1,567:-1,11602:-1

//=======================================================
//Moscovia
//=======================================================
morocc,169,105,0	shop	Cat Paw Shrimp Trader#morc	495,515:-1,567:-1,11602:-1

//=======================================================
// Payon
//=======================================================
payon,157,96,0	shop	Cat Paw Shrimp Trader#pay	495,515:-1,567:-1,11602:-1

//=======================================================
// Port Malaya
//=======================================================
malaya,286,175,0	shop	Cat Paw Shrimp Trader#ma	495,515:-1,567:-1,11602:-1

//=======================================================
// Prontera
//=======================================================
prontera,137,223,0	shop	Cat Paw Shrimp Trader#prt	495,515:-1,567:-1,11602:-1

//=======================================================
// Rachel
//=======================================================
ra_in01,243,272,0	shop	Cat Paw Shrimp Trader#ra	495,515:-1,567:-1,11602:-1

//=======================================================
//Umbala
//=======================================================
um_in,104,118,0	shop	Cat Paw Shrimp Trader#um	495,515:-1,567:-1,11602:-1

//=======================================================
//Veins
//=======================================================
ve_in,263,303,0	shop	Cat Paw Shrimp Trader#ve	495,515:-1,567:-1,11602:-1

//=======================================================
//Yuno
//=======================================================
yuno,205,116,0	shop	Cat Paw Shrimp Trader#yun	495,515:-1,567:-1,11602:-1
```

Only thing I cant test is the way they are facing because I don't have access to a korean game client. But this should at least help since all the locations match last weeks kro navigation. Prices in itemdb are correct to what iro has them selling for. Enjoy owo/

**Comment by Atemo** [comment]
First thanks for the base! Looking at the lua file I think

- (aldeba_in,89,56 is on aldebaran)
- alberta,111,159 is missing
- about **ayo_in01,15,177** : the map is ayothaya on the lua. The map should probably be ayo_in01 but the npc doesn't exist in. We will skip it for now.

However the NPCs uses a _npctalk attached to the player_ and the script command currently not implemented.
So far it should look like
```-	shop	doram_shrimp	-1, 515:-1, 567:-1, 11602:-1

izlude,117,165,3	script	Cat Paw Shrimp Merchant#iz	4_M_MERCAT1,{
	npctalk "Here are the shrimps, carrots and catnip!";	// npctalk should be attached to the player
	switch( select( "Buy", "Sell", "Cancel" ) ) {
	case 1:
		callshop "doram_shrimp",1;
		end;
	case 2:
		callshop "doram_shrimp",2;
		end;
	case 3:
		npctalk "Come back again soon!";					// npctalk should be attached to the player
		end;
	}
}

alberta,111,159,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#alb	4_M_MERCAT1
aldeba_in,89,56,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#ald	4_M_MERCAT1
amatsu,102,122,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#ama	4_M_MERCAT1
// ayo_in01,15,177
brasilis,249,257,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#bra	4_M_MERCAT1
comodo,103,136,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#com	4_M_MERCAT1
dewata,193,169,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#dew	4_M_MERCAT1
einbroch,164,316,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#ein	4_M_MERCAT1
ein_in01,194,22,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#ein1	4_M_MERCAT1
dic_in01,242,119,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#dic	4_M_MERCAT1
ecl_in01,72,86,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#ecl	4_M_MERCAT1
geffen_in,75,140,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#gef	4_M_MERCAT1
gonryun,147,88,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#gon	4_M_MERCAT1
hugel,76,165,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#hug	4_M_MERCAT1
izlude,117,165,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#iz	4_M_MERCAT1
izlude_a,117,165,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_a	4_M_MERCAT1
izlude_b,117,165,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_b	4_M_MERCAT1
izlude_c,117,165,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_c	4_M_MERCAT1
izlude_d,117,165,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Trader#iz_d	4_M_MERCAT1
lasagna,166,117,6	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#las	4_M_MERCAT1
lhz_in02,29,219,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#lhz	4_M_MERCAT1
lou_in02,241,168,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#lou	4_M_MERCAT1
malaya,286,175,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#mal	4_M_MERCAT1
mora,110,118,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#mora	4_M_MERCAT1
morocc,169,105,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#moro	4_M_MERCAT1
mosk_in,30,246,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#mosk	4_M_MERCAT1
payon,157,96,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#pay	4_M_MERCAT1
prontera,137,223,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#prt	4_M_MERCAT1
ra_in01,243,272,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#ra	4_M_MERCAT1
um_in,104,118,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#um	4_M_MERCAT1
ve_in,263,303,3	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#ve	4_M_MERCAT1
xmas_in,32,38,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#xma	4_M_MERCAT1
yuno,205,116,5	duplicate(Cat Paw Shrimp Trader#iz)	Cat Paw Shrimp Merchant#yu	4_M_MERCAT1
```

**Comment by TrueNoir** [comment]
From what i noticed in iro atleast the ones i could access where set up as
standard merchant shops. Whether its different on kro though not too sure
but regardless I was just trying to help out cause on my server people kept
asking for merchants for summoner class so i made it and decided to help
out rathena with it as well ^_^

**Comment by TrueNoir** [comment]
I tried to run the script you did and there's afew problems with it which I resolved as well as you also had 2 copies of the izlude,117,165 which you used for the npctalk so the server which should work. 

`izlude,117,165,3	script	Cat Paw Shrimp Merchant#iz	4_M_MERCAT1,{
	npctalk "Here are the shrimps, carrots and catnip!";	// npctalk should be attached to the player
	switch( select( "Buy", "Sell", "Cancel" ) ) {
	case 1:
		callshop "doram_shrimp",1;
		end;
	case 2:
		callshop "doram_shrimp",2;
		end;
	case 3:
		npctalk "Come back again soon!";					// npctalk should be attached to the player
		end;
	}
}

alberta,111,159,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#alb	4_M_MERCAT1
aldeba_in,89,56,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#ald	4_M_MERCAT1
amatsu,102,122,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#ama	4_M_MERCAT1
// ayo_in01,15,177
brasilis,249,257,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#bra	4_M_MERCAT1
comodo,103,136,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#com	4_M_MERCAT1
dewata,193,169,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#dew	4_M_MERCAT1
einbroch,164,316,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#ein	4_M_MERCAT1
ein_in01,194,22,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#ein1	4_M_MERCAT1
dic_in01,242,119,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#dic	4_M_MERCAT1
ecl_in01,72,86,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#ecl	4_M_MERCAT1
geffen_in,75,140,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#gef	4_M_MERCAT1
gonryun,147,88,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#gon	4_M_MERCAT1
hugel,76,165,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#hug	4_M_MERCAT1
izlude_a,117,165,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#iz_a	4_M_MERCAT1
izlude_b,117,165,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#iz_b	4_M_MERCAT1
izlude_c,117,165,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#iz_c	4_M_MERCAT1
izlude_d,117,165,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#iz_d	4_M_MERCAT1
lasagna,166,117,6	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#las	4_M_MERCAT1
lhz_in02,29,219,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#lhz	4_M_MERCAT1
lou_in02,241,168,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#lou	4_M_MERCAT1
malaya,286,175,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#mal	4_M_MERCAT1
mora,110,118,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#mora	4_M_MERCAT1
morocc,169,105,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#moro	4_M_MERCAT1
mosk_in,30,246,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#mosk	4_M_MERCAT1
payon,157,96,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#pay	4_M_MERCAT1
prontera,137,223,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#prt	4_M_MERCAT1
ra_in01,243,272,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#ra	4_M_MERCAT1
um_in,104,118,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#um	4_M_MERCAT1
ve_in,263,303,3	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#ve	4_M_MERCAT1
xmas_in,32,38,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#xma	4_M_MERCAT1
yuno,205,116,5	duplicate(Cat Paw Shrimp Merchant#iz)	Cat Paw Shrimp Merchant#yu	4_M_MERCAT1`
^ this won't error

**Comment by Akkarinage** [comment]
If these are all correct, could they be throw into a PR and merged straight into master?

**Comment by Lemongrass3110** [comment]
I think they were merged into #2105 already

**Comment by aleos89** [comment]
Fixed in a5afb06.
